linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/ansible/shared.yml

```diff
@@ -13,12 +13,20 @@
     state: present
 {{% endif %}}
 
-- name: "{{{ rule_title }}}"
+- name: "{{{ rule_title }}} - Set Crypto Policy"
   ansible.builtin.lineinfile:
     path: /etc/crypto-policies/config
     regexp: '^(?!#)(\S+)$'
     line: "{{ var_system_crypto_policy }}"
     create: yes
 
-- name: Verify that Crypto Policy is Set (runtime)
+- name: "{{{ rule_title }}} - Check current crypto policy (runtime)"
+  ansible.builtin.command: /usr/bin/update-crypto-policies --show
```

Comment:
This will show the state after modifying the /etc/crypto-policies/config by the previous task. This causes that the next task that runs `--set` will never be executed.

I suggest removing the task that modifies `/etc/crypto-policies/config` above.